src/App.js

```diff
@@ -459,6 +464,10 @@ function handleParentWindowMessage(event) {
     case "frameShown":
       handleFrameShownEvent();
       break;
+    case "listContacts":
+    console.log("LISTING CONTACTS");
```

Comment:
Get rid of these extra log lines

Reply:
fixed, commenting here so i can keep track of what i have left to do.